modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,296 @@
+
+| `imagetools_cors_hosts`             | `editimage_cors_hosts`           | Option                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
+
+| `imagetools_credentials_hosts`      | `editimage_credentials_hosts`    | Option                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
+
+| `imagetools_fetch_image`            | `editimage_fetch_iamge`          | Option                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
+
+| `imagetools_api_key`                | `editimage_api_key`              | Option                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
+
+| `images_upload_timeout`             | `editimage_upload_timeout`       | Option                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
+
+| `editimage_proxy`                   | `export_image_proxy_service_url` | Option                              | Renamed for easier configuration when used with https://tiny.cloud/docs/enterprise/server/[Tiny services].                                                                                                                                                                                                    |
```

Comment:
I don't know what I'm talking about here, but this seems odd to me? @lnewson @TheSpyder Could one of you please confirm if this line is correct?

Reply:
Yeah, it was renamed it was a new option that makes it easier to work with the backend services we provide. I'd suggest having a look at the `/tinymce/6/editimage/#editimage_proxy_service_url` page/option which might provide more details.

Reply:
```suggestion
| `imagetools_proxy`             | `editimage_proxy`       | Option                              | NB: _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                   | see xref:new-and-improved-plugins-imagetools[EditImage]
```